RDKEMW-3584: Set dcm config when loading profile from Disk

shivabhaskar opened the pull request:
Reason for change: the default behaviour of dcm implementation is to process the config from previous boot if the xconf communication is not done bringing the similar implementation based on the discussion
Test Procedure: dcm-agent config should be processed when previous boot config is available
Risks: Low

Ananth916 committed:
RDK-55195: Adding RCVRY cert for rdkcertselector

Reason for change: rdkcertselector init for RCVRY certficate
Test Procedure: None
Risks: Low

Ananth916 committed:
Update curlinterface.c

Fixing format errors

Ananth916 committed:
Update curlinterface.c

Debugging purpose

Ananth916 committed:
Update curlinterface.c

Debugging purpose

Ananth916 committed:
Update curlinterface.c

Debugging purpose

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

Ananth916 committed:
Update curlinterface.c

shivabhaskar committed:
RDKEMW-3584: Set dcm config when loading profile from Disk

Reason for change: the default behaviour of dcm implementation is to process the config from previous boot if the xconf communication is not done bringing the similar implementation based on the discussion
Test Procedure: dcm-agent config should be processed when previous boot config is available
Risks: Low

rdkcmf-jenkins commented:
<!-- Comment managed by coverity-report-output-v7 action, do not modify!
7d22d56a6d7036fc2981066817766ae4
PRESENT
-->

# Coverity Issue - Check of thread-shared field evades lock acquisition
Thread1 sets "initialized" to a new value. Now the two threads have an inconsistent view of "initialized" and updates to fields correlated with "initialized" may be lost.

_High Impact, CWE-543_
_LOCK_EVASION_

## How to fix
 Guard the modification of "initialized" and the read used to decide whether to modify "initialized" with the same set of locks.

## Issue location
This issue was discovered outside the diff for this Pull Request. You can find it at:
[source/bulkdata/profilexconf.c:703](undefined//blob/undefined/source/bulkdata/profilexconf.c#L703)

Abinesh44-Devi committed:
Add MTLS support for T2 in Pod (#106)

* Add MTLS support for T2 in Podd

* Add MTLS support for T2 in Podd

* Add MTLS support for T2 in Podd

Commit:
RDK-57209 : Optimize rdkcertselector implementation in telemetry

Reason for change: Optimize the certselector invoke in T2
Test Procedure: Validate T2 profile report
Risks: Low

Signed-off-by: plaksh175_comcast <PiramanayagamMoneka_Lakshmi@comcast.com>

Ananth916 committed:
RDK-57209 : Optimize rdkcertselector implementation in telemetry

Reason for change: Optimize the certselector invoke in T2
Test Procedure: Validate T2 profile report
Risks: Low

Ananth916 committed:
RDK-57209 : Optimize rdkcertselector implementation in telemetry

Reason for change: Optimize the certselector invoke in T2
Test Procedure: Validate T2 profile report
Risks: Low

Ananth916 committed:
RDK-57209 : Optimize rdkcertselector implementation in telemetry

Reason for change: Optimize the certselector invoke in T2
Test Procedure: Validate T2 profile report
Risks: Low

Ananth916 committed:
RDK-57209 : Optimize rdkcertselector implementation in telemetry

Reason for change: Optimize the certselector invoke in T2
Test Procedure: Validate T2 profile report
Risks: Low

shibu-kv committed:
Changelog updates for release 1.4.4

shivabhaskar committed:
RDK-57946 : Debug Build

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/telemetry?pullRequest=103) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/telemetry?pullRequest=103) before we can accept your contribution.<br/>**4** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: shivabhaskar<br/>:white_check_mark: Ananth916<br/>:white_check_mark: Abinesh44-Devi<br/>:white_check_mark: shibu-kv<br/>:x: plaksh175_comcast<br/><hr/>**plaksh175_comcast** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/telemetry?pullRequest=103) it.</sub>
